backend/src/v2/compiler/argocompiler/container.go

```diff
@@ -283,6 +286,59 @@ func (c *workflowCompiler) addContainerExecutorTemplate(refName string) string {
 			extendPodMetadata(&executor.Metadata, k8sExecCfg)
 		}
 	}
+	caBundleCfgMapName := os.Getenv("ARTIFACT_COPY_STEP_CABUNDLE_CONFIGMAP_NAME")
+	caBundleCfgMapKey := os.Getenv("ARTIFACT_COPY_STEP_CABUNDLE_CONFIGMAP_KEY")
+	caBundleMountPath := os.Getenv("ARTIFACT_COPY_STEP_CABUNDLE_MOUNTPATH")
```

Comment:
We initially picked these based on the kfp-tekton implementation, but I think we can pick better env vars now that are more generic, I propose: 

`EXECUTOR_CABUNDLE_CONFIGMAP_NAME`
`EXECUTOR_CABUNDLE_CONFIGMAP_KEY`
`EXECUTOR_CABUNDLE_MOUNTPATH`

since these are

Reply:
Done, updated to have these new env vars.